### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by er-contreras**
# Set menu 

- We learn how to use a linter on JS.
- Also we implement the DOM to our menu to open.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,11 +33,18 @@ <h1>John Doe</h1>
...
+    <nav class="menu-dropdown flex">
+      <div class="flex">
+        <button type="button" class="close-btn" onclick="closeMenu()"><i class="fas fa-times fa-lg"></i></button>
+      </div>
+      <ul>
+        <li onclick="closeMenu()">Portfolio<i class="fas fa-chevron-right fa-xs"></i></li>
+        <li onclick="closeMenu()">About<i class="fas fa-chevron-right fa-xs"></i></li>
+        <li onclick="closeMenu()">Contact<i class="fas fa-chevron-right fa-xs"></i></li>
+      </ul>
     </nav>
```
Comment (Temmarie) [inline index.html]: - [x] Good job implementing the dropdown menu as requested! However, there are a few tweaks required for you to make in this section:
- The menu items should link to actual parts of the page. See [here](https://www.w3schools.com/html/html_links.asp) for information on how to link to items on a page.
- You have some background image repetition and misplacement, kindly fix it.
- The headline also has some issues with the background image decoration. Kindly refer to the Figma design template and make these corrections.

_See images below for reference._

**Original** | **Your clone**
  ----------- | -----------
  image | image
image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @eri8-9     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 
- The dropdown menu works as expected ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
- [ ] According to the project's requirements, you should set up both JavaScript and HTML&CSS linters in the project repository. Let this change reflect in the `linters.yml` file.

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: index.html (hunk 1 above)
Comment (er-contreras) [reply index.html]: Thanks!

**Review by MahmoudBakr23 — APPROVED** [review]
Hi, @eri8-9 and @jchernandez87 😃, 

Great job making the required changes by the previous code reviewer 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**OPTIONAL**

- The same way you managed the navigation through the items' links, you can do it to the icons `>` that belong to each one of them, because it's just closing the menu!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**